packages/expo-updates/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### 🎉 New features
 
+- Updated `@expo/metro-config`. ([#12252](https://github.com/expo/expo/pull/12252) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
would be helpful to describe the version upgrade and anything meaningful that's new